src/arithmetic.jl

```diff
@@ -126,9 +126,12 @@ end
 function remainder_product(a, b, aux, Δnegl)
     Δa = a.pol(aux)
     Δb = b.pol(aux)
-    Δ = Δnegl + Δb * a.rem + Δa * b.rem + a.rem * b.rem
+    Δ₁ = Δa * b.rem + a.rem * (Δb + b.rem)
+    Δ₂ = Δb * a.rem + b.rem * (Δa + a.rem)
+    Δ = Δnegl + (Δ₁ ∩ Δ₂)
```

Comment:
I have one doubt, should't this be `Δ = (Δnegl + Δ₁) ∩ (Δnegl + Δ₂)` ? Assuming that `Δnegl` is the bound on `s(D − x0)` using Bunger's notation.

Reply:
I  think this should be the same, but maybe I am wrong; do you have a case in mind where the equality shouldn't hold?

Indeed, Δnegl corresponds to `s(D-x0)`, see e.g. [this line](https://github.com/JuliaIntervals/TaylorModels.jl/blob/d7070c0e3b497a480eb7babc331c0eb87e7d26c5/src/arithmetic.jl#L96). The odd naming of variables is due to me, but to my defense, I worked out before Bünger's paper.

Reply:
> do you have a case in mind where the equality shouldn't hold?

If `X`, `A` and `B` are sets such that `A` and `B` are disjoint. For example:

```julia
A = 0.2 .. 0.3
B = 0.4 .. 0.5
X = 1.0 .. 2.0

@show X + (A ∩ B)
@show (X + A) ∩ (X + B)

X + A ∩ B = ∅
(X + A) ∩ (X + B) = [1.39999, 2.30001]
```

The semantics is

```
X+(A∩B)={y:y=x+c,x∈X,c∈A and c∈B}
(X+A)∩(X+B)={y:y=x+c,x∈X,c∈A and y=x′+c′,x′∈X,c′∈B}
```

Reply:
Good catch indeed!!